apps/vault/package.json

```diff
@@ -40,6 +41,6 @@
     "truffle-hdwallet-provider": "0.0.3"
   },
   "dependencies": {
-    "@aragon/os": "^3.1.1"
+    "@aragon/os": "^3.1.9"
   }
```

Comment:
To consider: we should either update the version to this package to be `1.0.1` or `3.0.0`.

I prefer `1.0.1` for the similar interface, despite the opportunity to line up the versions with our `arapp.json` file, which we'd need to increment as well to `3.0.0` to deploy to rinkeby.

Reply:
I imagine you are referring to `@aragon/apps-vault`'s version and not aOS. If so, I support keeping it in version `1.0.1` or `1.1.0` given the nature of the change, even though we didn't break the API.

Reply:
I think this could be one of the reasons the CI is failing, in Finance the vault dependency was set to `^2.0.1` and the vault we had in the repo was `1.1.0`.  The issue could be that given that the local version is not able to satisfy the vault dependency, it lerna then tries to fetch it from NPM. This is what it looks like it is happening in the CI as it keeps complaining about the solidity issue we solved here: https://github.com/aragon/aragon-apps/pull/292/commits/3757ec1d526a65cb73b87de8a0ea8bc9e77187b7.

There is another issue with setting the vault version to `1.1.0`, because `2.0.1` is already published in NPM https://www.npmjs.com/package/@aragon/apps-vault and we won't be able to overwrite it when the time comes.

I propose jumping to vault `3.0.0` given that we ended up breaking the API.